### PR TITLE
Small code improvements.

### PR DESCRIPTION
**Pull request by kdzwinel**
Just couple of small improvements - nothing major. Maybe you'll find it useful.

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
-(function() {
+(function () {
 
-  var selector = 'span.renderable-component-text-box-content';
-    selector += ', span.renderable-component-text';
-
-  if (window.location.href === 'https://www.google.com/maps')
+  if (window.location.href === 'https://www.google.com/maps') {
```
Comment (kdzwinel) [inline src/core.js]: If we are redirecting user, then it should be the first thing we will do.

File: src/core.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
     window.location.href = 'https://www.google.com.br/maps';
+    return;
```
Comment (kdzwinel) [inline src/core.js]: Since we are redirecting, we don't have to do anything else.

File: src/core.js [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
 
   $(document).arrive(selector, function () {
-    var code = $(this);
-    code.each(getRoute);
+    $(this).each(getRoute);
   });
 
-  function getRoute(index) {
+  function getRoute() {
+    var codigo = $(this).text();
```
Comment (kdzwinel) [inline src/core.js]: We are referring to `$(this).text()` a lot, it's better to cache it.

File: src/core.js [hunk 4]
```diff
@@ -1,22 +1,25 @@
...
+
     for (var i = 0, len = linhasGrandeRecife.length; i < len; i++) {
-      if (linhasGrandeRecife[i].codigo !== $(this).text())
-        continue;
-      $(this).text(linhasGrandeRecife[i].linha);
-        break;
+      if (linhasGrandeRecife[i].codigo === codigo) {
+        $(this).text(linhasGrandeRecife[i].linha);
+        return;
```
Comment (kdzwinel) [inline src/core.js]: Whenever we will find a match, we can return from this function.

**Comment by patrickrbc** [comment]
Good changes, thank you!

**Pull request merged**